src/app/pages/apps/components/installed-apps/app-info-card/app-info-card.component.ts

```diff
@@ -163,4 +169,14 @@ export class AppInfoCardComponent {
       });
     });
   }
+
+  rollbackApp(): void {
+    this.matDialog.open(AppRollbackModalComponent, { data: this.app })
+      .afterClosed()
+      .pipe(filter(Boolean), untilDestroyed(this))
+      .subscribe(() => {
+        this.app.history = null;
```

Comment:
@undsoft 
refetching data won't help I think 
cuz I may close modal and it will be on the same state

From my point of view we need to immediately hide the button if Roll Back process started.

Reply:
Isn't rolling back a job?

Reply:
it's a job

Reply:
Isn't card being updated by events when something happens to a chart? Alternatively, if the issue with reloading is with possibility of user closing modal without rolling back, I'd rather do that over mutation.

Reply:
I tested, when job is complete - chart still has ROLLBACK button, after page reload - there is no button anymore.